Add new How to support us landing page

matyikriszta opened the pull request:
### Description
This PR adds a new How to support us landing page to the site. It adds the Volunteer with us and Donate sections to the page, as well as the title. I haven't added the page to the navigation yet. I figured we'd keep it semi-hidden while working on it (e.g. not accessible from the navigation).

NOTE: @KimberleyCook I need your help with sourcing assets for this page, do you have a better picture for any of the cards in the Volunteer with us section? Also let me know if you want to change any of the copy, or we can launch with this and always update in the future. Also, for now the styling is pretty bare bones, I want to get most of the content on the page and then I'm going to spend some time making it look better/more exciting.

### Status
Ready for Review

### Screenshots
| Mobile | Desktop |
| --- | --- |
| ![Screenshot 2024-02-15 at 22-32-04 codebar](https://github.com/codebar/planner/assets/5873816/8090e4e7-c027-4946-8c3a-8e1f19ba2d97) | ![Screenshot 2024-02-15 at 22-30-03 codebar](https://github.com/codebar/planner/assets/5873816/c78874d4-1dd2-4d02-bc0a-c6e7f9e4010a) |

KimberleyCook commented on config/locales/en.yml:
```diff
@@ -686,6 +686,18 @@ en:
+        description: Affect change as a volunteer coach with codebar - your expertise and guidance will pave the way for aspiring developers, unlocking the world of coding and creating opportunities for folks from groups under represented in the tech industry.
+      volunteer:
+        title: Other skills
+        description: codebar welcomes volunteers with a diverse range of skills - whether it's event coordination, fundraising, marketing or any other skill, your unique abilities can enrich our community and drive our mission forward.
```
I think this should read as 2 sentences. Like so, `codebar welcomes volunteers with a diverse range of skills - whether it's event coordination, fundraising, marketing or any other skill. Your unique abilities can enrich our community and drive our mission forward.`

matyikriszta commented:
@KimberleyCook I updated the copy and added some of the images you sent me, I think they look really good!

![Screenshot 2024-03-18 at 19-57-36 codebar](https://github.com/codebar/planner/assets/5873816/231ea66b-879e-4f5d-827e-89b0db626441)

KimberleyCook reviewed (approved):
Looks banging 🎉🎉🎉🎉🎉 nice work
